backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -57,5 +57,11 @@ SearchException conflictingFieldFormattersForSearch(String absoluteFieldPath,
 			+ " Given predicate was: '%1$s'" )
 	SearchException cannotMixElasticsearchSearchQueryWithOtherPredicates(SearchPredicate predicate);
 
+	@Message(id = 9, value = "Field '%1$s' is not an object field in index '%2$s'." )
+	SearchException nonObjectFieldForNestedQuery(String absoluteFieldPath, String indexName);
```

Comment:
So it's probably just me but usually I try to be consistent with having the more general notions first, even if it doesn't fit the order of the message. This way it's all consistent.

Reply:
So, `nestedQueryNonObjectField`?

Reply:
Ah no sorry if I wasn't clear but I was talking about the order of the parameters.